docs/reference/modules/scripting.asciidoc

```diff
@@ -24,23 +24,18 @@ and give the most flexibility.
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Language
-    |Sandboxed
     |Required plugin
```

Comment:
I think i recommend against the DOS example. The way i see it, that is a feature to try to prevent runaway scripts to help the system be stable. But I don't think we should imply you can put ES on the internet and you wont' be DOSed. I can DOS the hell out of any open ES system, and I don't need scripting to do it.